src/commands/deploy/latest.js

```diff
@@ -283,7 +282,8 @@ export default async function main(
         meta
       },
       {
-        name: argv['--name'] || localConfig.name || filesName
+        project,
+        name: project
```

Comment:
Here we set the project as the `name`. Due to backward compatibility of our backend, we still need to `name` field. But if the `project` field is there it'll use that instead.

Reply:
👍